### DISCUSSION TITLE
Can bus and cranking starter relay

### DISCUSSION THREAD

**Discussion by monalos** [cedricp/ddt4all#1200] [General]
Hello, I want to operate the starter relay from the can bus. When I send a command to Canbus, the starter relay will work and the car will start. I found something like in the image, can you help? Which address should I read and what should I send? I will send this command after the ignition is turned on. I can turn on the ignition but I'm looking for the starter command.

The cranking relay is connected to BCM, BCM cranks it, there is a situation regarding the starter relay on off at the 55D can bus address, when I look at the bcm with ddt4all can sniffer.

**Comment by monalos** [comment]
![Uploading IMG_9095.png…]()

**Comment by matzel82** [comment]
This wants never work. You can’t connect a relay with Bus. The current is to low and the signal wants make all ECUs an error.

**Comment by monalos** [comment]
I will not connect the relay to can bus. I want to operate the starter relay with a command that I will send to the can bus. There is something like this in the ECM: When I press the start button, cranking is allowed and the car starts, but when I press the remote start button, it says no request, the ignition is on, the car does not start. Can I send the authorization code to CanBusa for this cranking authorization? I am trying to do the original remote start, when I press the remote start button, the ignition turns on, but it waits and there is no cranking. If I energize the manual starter relay when the ignition is turned on, the car starts remotely.
![IMG_9120](https://github.com/user-attachments/assets/2aed4bdc-d0d2-43ef-b5c4-a36c1278229d)

@matzel82

**Reply by monalos** [reply]
When I want to start it remotely, the ignition turns on and it looks like the video. Pay attention, it says "no request 1 second", in the cranking not allowed section.
